templates/README.md

```diff
@@ -12,6 +12,13 @@ The documents declared in this repository are:
 
 - ...
 
+## Deployment
```

Comment:
My first thought was that settings were more to be put after the configuration of the whole repo because it would be lost after init

Let me know what you prefer then

Reply:
I feel that the README of the resulting repo will be targeted at contributors and reusers of the declarations, who will not be concerned with continuous deployment of the updates.